docker-compose.yml

```diff
@@ -196,8 +204,17 @@ services:
       PGBOUNCER_SERVICE_HOST: coreapi-pgbouncer
       WORKER_ADMINISTRATION_REGION: ingestion
       WORKER_RUN_DB_MIGRATIONS: "1"
+      MAX_COMPANION_PACKAGES: 5
+      MAX_ALTERNATE_PACKAGES: 2
+      OUTLIER_THRESHOLD: 0.6
+      UNKNOWN_PACKAGES_THRESHOLD: 0.3
+      PGM_SERVICE_HOST: "kronos-stack-analysis.dev.rdu2c.fabric8.io"
+      LICENSE_SERVICE_HOST: "stack-license-stack-license.dev.rdu2c.fabric8.io"
+      PGM_SERVICE_PORT: "80"
+      LICENSE_SERVICE_PORT: "80"
   anitya-server:
-    image: registry.devshift.net/slavek/anitya-server
+    #image: registry.devshift.net/slavek/anitya-server
+    image: slavek/anitya-server
```

Comment:
@jpopelka, I have no idea how that went through. Must be a merge issue. I will likely close this against a new PR. Thanks.